PYME/Analysis/points/DeClump/deClump.c

```diff
@@ -977,6 +989,10 @@ static PyObject * aggregateSum(PyObject *self, PyObject *args, PyObject *keywds)
         return NULL;
     }
 
+        //initialise memory in output arrays
+    PyArray_FILLWBYTE(outVarA, 0);
+    //PyArray_FILLWBYTE(outSigA, 0);
```

Comment:
no reason to add the commented out line, but otherwise I'd say go ahead and merge @David-Baddeley